fs/cromfs/cromfs.h

```diff
@@ -26,6 +26,7 @@
  ****************************************************************************/
 
 #include <sys/types.h>
+#include <nuttx/compiler.h>
```

Comment:
```suggestion
#include <nuttx/compiler.h>
```
I think `stdint.h` already contains `include <nuttx/compiler.h>`